.github/workflows/package-python.yml

```diff
@@ -0,0 +1,80 @@
+name: Create Python Package
```

Comment:
The "package-python" name is a little misleading... at the very least, it's "Python-package"; but if it's a beeware project, it's going to be Python regardless.

Reply:
That's definitely fair.

With that, GitHub currently has a limitation requiring all workflows to exist in the `.github/workflows/` directory. So, while we may not end up with too many workflows, I was looking for a way for workflows related to the same thing/tool/concept to alphabetically. That's why I started the workflows for `pre-commit` with `pre-commit` and same with `towncrier`. This leads, though, to a bit of awkward grammar with the action word coming last.

With that in mind, perhaps `package-beeware-create.yml` best follows those rules (although equally awkward)....and perhaps where a workflow like `package-beeware-deploy.yml` could theoretically exist alongside it...

Reply:
Landed on `python-package-create.yml`.